aws/values.tf

```diff
@@ -0,0 +1,47 @@
+variable "aws_region" {
+  default = "eu-west-1"
+}
+
+variable "vpc_id" {}
+
+variable "vpc_subnet_ids" {
+  type = "list"
+}
+
+variable "vpc_subnet_id_instance" {}
+
+variable "instance_availability_zone" {}
```

Comment:
`availability_zone` Also please add meaningful descriptions like that subnetid should belong to the chosen zone and so.